meteoroid/core/models.py

```diff
@@ -40,8 +40,10 @@ def get_dict(self):
         }
 
     def equals_faas_data(self, faas_data):
-        function = Function.objects.filter(fiware_service=self.fiware_service,
-                                           fiware_service_path=self.fiware_service_path).get(name=faas_data['action_name'])
+        function = Function.objects.filter(
+            fiware_service=self.fiware_service,
+            fiware_service_path=self.fiware_service_path).get(
+            name=faas_data['action_name'])
```

Comment:
I think L46 needs four more spaces than L45.
The `name` parameter may be mistaken for the `Function.objects.filter` method parameter in spite of the `get` method parameter.

Reply:
I used autopep8.
That is a good idea. I updated.